packages/lesswrong/components/messaging/NewConversationDialog.tsx

```diff
@@ -115,7 +129,25 @@ const styles = (theme: ThemeType): JssStyles => ({
     width: 20,
     height: 20,
     cursor: "pointer",
-  }
+  },
+  chip: {
+    margin: 4,
+    backgroundColor: theme.palette.background.primarySlightlyDim,
+  },
+  submitButton: {
+    padding: 8,
+    fontSize: "16px",
+    minWidth: 28,
+    minHeight: 28,
+    marginLeft: "auto",
+    backgroundColor: "transparent",
+    color: theme.palette.primary.main,
+    background: "transparent",
+    fontWeight: 500,
+    '&:hover': {
+      backgroundColor: theme.palette.background.primaryDim,
+    }
```

Comment:
I made this match the one in the message box (see https://github.com/ForumMagnum/ForumMagnum/pull/8614#issuecomment-1892006201 for context on why I added this)